client/js/components/user.jsx

```diff
@@ -0,0 +1,74 @@
+var React = require("react");
+var c3 = require("c3");
+
+var User = React.createClass({
+  componentDidMount: function() {
+    var chart = c3.generate({
+      bindto: ".contribution-grade",
+      data: {
+        columns: [["data", 0]],
+        type: "gauge",
+        onclick: function(d, i) {
+          console.log("onclick", d, i);
+        },
+        onmouseover: function(d, i) {
+          console.log("onmouseover", d, i);
+        },
+        onmouseout: function(d, i) {
+          console.log("onmouseout", d, i);
+        }
+      },
+      gauge: {
+        label: {
+          format: function(value, ratio) {
+            return value;
+          },
+          show: true
+        },
+        min: 0,
+        max: 6,
+        units: "schemas",
+        width: 39
+      },
+      color: {
+        pattern: ["#FF0000", "#F97600", "#F6C600", "#60B044"],
+        threshold: {
+          unit: "value"
+        }
+      },
+      size: {
+        height: 150
```

Comment:
hmm... I think because the property names describe what the numbers are they are not really magic numbers.